shakedown/dcos/package.py

```diff
@@ -74,6 +74,12 @@ def install_package(
 
     cosmos = _get_cosmos()
     pkg = cosmos.get_package_version(package_name, package_version)
+    if app_id is None:
+        properties = pkg.config_json().get('properties')
+        if 'service' in properties:
+            app_id = properties['service']['properties']['name']['default']
```

Comment:
there is a danger here... this is not defined as a standard universe package.     This works for marathon.